modules/nodes-nodes-resources-configuring-auto.adoc

```diff
@@ -74,7 +74,7 @@ spec:
 ----
 <1> Assign a name to CR.
 <2> Add the `autoSizingReserved` parameter set to `true` to allow {product-title} to automatically determine and allocate the `system-reserved` resources on the nodes associated with the specified label. To disable automatic allocation on those nodes, set this parameter to `false`.
-<3> Specify the label from the machine config pool that you configured in the "Prerequisites" section. You can choose any desired labels for the machine config pool, such as `custom-kubelet: small-pods`, or the default label, `pools.operator.machineconfiguration.openshift.io/worker: ""`.
```

Comment:
I think this selector is unrelated, so what is here still applies as before.

This is a label selecting a kubeletconfig for a pool, whereas the issue in the description happens for labels selection machineconfigs for a pool (which is not the same thing)